### PR TITLE
[IOTDB-635]Added workaround when doing Aggregation over boolean Series.

### PR DESCRIPTION
**Pull request by JulianFeinauer**
As discussed, please have a look @jixuan1989

### CODE REVIEW

File: grafana/src/main/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImpl.java [hunk 1]
```diff
@@ -98,8 +98,28 @@ public Object doInConnection(Connection connection) throws SQLException {
...
+    try {
+      return querySeriesInternal(s, timeRange, function);
+    } catch (Exception e) {
+      logger.info("Execution failed, trying now with FIRST Function!");
+      // Try it with FIRST
+      try {
+        return querySeriesInternal(s, timeRange, "FIRST");
+      } catch (Exception e2) {
+        logger.warn("Even FIRST query did not succeed, returning NULL now", e2);
+        return null;
```
Comment (jixuan1989) [inline grafana/src/main/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImpl.java:117]: how about empty List?

File: grafana/src/main/resources/application.properties [hunk 2]
```diff
@@ -32,4 +32,4 @@ isDownSampling=true
 interval=1m
 # aggregation function to use to downsampling the data
 # COUNT, FIRST_VALUE, LAST_VALUE, MAX_TIME, MAX_VALUE, AVG, MIN_TIME, MIN_VALUE, NOW, SUM
-function=avg
\ No newline at end of file
+function=AVG
```
Comment (jixuan1989) [inline grafana/src/main/resources/application.properties:35]: add a comment here: if the data type does not support AVG, then FIRST will be used.

**Comment by JulianFeinauer** [comment]
Makes sense I guess. Will add that

Holen Sie sich Outlook für Android<https://aka.ms/ghei36>

________________________________

File: grafana/src/main/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImpl.java (hunk 1 above)
Comment (JulianFeinauer) [reply grafana/src/main/java/org/apache/iotdb/web/grafana/dao/impl/BasicDaoImpl.java]: +1, done

File: grafana/src/main/resources/application.properties (hunk 2 above)
Comment (JulianFeinauer) [reply grafana/src/main/resources/application.properties:36]: done.

**Pull request merged**